servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -509,6 +509,7 @@ boolean processRole(ObjectStoreConnection con, Role originalRole, String domainN
         } else {
             // carrying over auditEnabled from original role
             role.setAuditEnabled(originalRole.getAuditEnabled());
+            mergeOriginalRoleAndMetaRoleAttributes(originalRole, role);
```

Comment:
we cannot change the behavior of the processRole - that method is used in lots of places.
this call must be done only in the method where templates are being handled.

Reply:
Done.